### ISSUE TITLE
REGS probtotext

### ISSUE THREAD

**Issue by falgreen** [CLINDA-AAU/hemaClass#42]
Super cool function and I really like the colour match in Patient summaries. 

Perhaps it can be misunderstood in relation to REGS when the probability is low. For instance a subject that is predicted to be sensitive with great confidence, will be classified as Sensitive with an extremely low probability. That could be interpreted as the subject has a low probability of being sensitive.

Should we do something about this?

**Comment by AEBilgrau** [comment]
Good to hear. Yeah, the argument `symmetric` of `probToText` was made as an attempt to handle that. This flips probabilities less than 0.5 and refers to 0.023, say, as an very high probability. I.e.:

``` R
probToText(0.023, symmetric = TRUE)
#[1] "very high probability (p = 0.02)"
```

It's not perfect. But I can't come up with something better. It should still match what is given in results table.

**Comment by AEBilgrau** [comment]
Obviously, the approach only makes sense for the dichotomous classifiers. Other suggestions are very welcome.

**Comment by falgreen** [comment]
I thinks the solution works well for Non REGS classifiers. And I agree that it is hard to come up with something good for REGS.

We could perhaps write something like:
For Sensitive (or intermediate): predicted to be sensitive with very low probability (p = 0.02) of being resistant.

and if resistant:  predicted to be resistant with very high probability (p = 0.98).

**Comment by AEBilgrau** [comment]
OK, I now follow that formulation. However, I guess this doesn't work for the Rituximap classifier with lytic/static stuff. 
But I guess the probabilities here are always "high" as the probability given is the largest (like BAGS)?  So the formulation:

"predicted to be <class> with <very high probability> (p = <0.99>)" always works?

**Comment by falgreen** [comment]
Exactly, it should neither be used for melphalan as far as I remember

> Den 26. okt. 2015 kl. 10.23 skrev Anders Ellern Bilgrau notifications@github.com:
> 
> OK, I now follow that formulation. However, I guess this doesn't work for the Rituximap classifier with lytic/static stuff. 
> But I guess the probabilities here are always "high" as the probability given is the largest (like BAGS)?. So the formulation:
> 
> "predicted to be <class> with <very high probability> (p = <0.99>)" always works?
>
